src/util/__tests__/copyTextToClipboard.test.ts

```diff
@@ -0,0 +1,25 @@
+import { copyTextToClipboard } from '../copyTextToClipboard';
+
+describe('copyTextToClipboard', () => {
+  it('should be defined', () => {
+    expect(copyTextToClipboard).toBeDefined();
+  });
+
+  it('should call navigator.clipboard.writeText with `test`', async () => {
```

Comment:
Could you add a test for a scenario where this fails? E.g. with `Promise.reject()`